### ISSUE TITLE
Fixer return used in PSR2 in error

### ISSUE THREAD

**Issue by Dragonrun1** [FriendsOfPHP/PHP-CS-Fixer#1018]
I've read through both PSR-1 and PSR-2 and no where do they say there should be a blank line before the return statement but is include in fixes when using ->level(Symfony\CS\FixerInterface::PSR2_LEVEL)  in config file or using the level setting on CLI.

**Comment by keradus** [comment]
The `return` fixer should not be applied on PSR2 level.
Which version of PHP CS Fixer are you using? Do you use `.php_cs` file ?

**Comment by Dragonrun1** [comment]
I was using 1.4.2 which did seem to have the problem in all cases but I've switched to 1.5@DEV and it seems to be working correctly.

Additional I figured out part of the problem was I misread the docs and was trying to use a .php_cs.dist file which it doesn't look for. Once I renamed the config file is it working correctly so it was simply using symfony rules because of lack of config file which is what is expected but was confusing ;) Maybe adding to the verbose output something which states no config file found and using defaults might be a good idea. The way I figured it out was by using the --config-file=".php_cs.dist" option and seeing the 'Loading config from ...' message. If you don't know it should show that it's easy to be confused why your config file doesn't seem to work. Since it looks like it was mostly something on my end and 1.5@DEV seems to be working for me I'm fine with you close out the issue.
